staging/src/slime.io/slime/modules/meshregistry/pkg/bootstrap/args.go

```diff
@@ -166,6 +166,8 @@ type NacosSourceArgs struct {
 	// username and password for nacos auth
 	Username string
 	Password string
+	// fetch services from all namespaces
+	AllNamespaces bool
```

Comment:
add comment for not supporting watching mode

Reply:
do it in another task, maybe cleancode.